pydicom/values.py

```diff
@@ -379,20 +379,18 @@ def convert_value(VR, raw_data_element, encoding=default_encoding):
         logger.debug('unable to translate tag %s with VR %s'
                      % (raw_data_element.tag, VR))
 
-        for convert_vr in convert_retry_VR_order:
-            vr = convert_vr
-            converter = converters[vr]
+        for vr in convert_retry_VR_order:
             if vr == VR:
                 continue
             try:
                 value = convert_value(vr, raw_data_element, encoding)
-                break
```

Comment:
This is just bad code on my part.  The break here is meant to say "Yeah!  we found a VR that converted this without error, let's move on".  The exception clause is meant to say "Oh, too bad, let's try the next one."